changes to log4j.properties don't have any effect?

I tried modifying `./src/main/resources/log4j.properties` but it doesn't seem to have any effect - I read somewhere that there is an slf4j -> log4j binding/adapter, is there something else I need to do other than modify the log4j.properties in order to get SimianArmy logging to a file?

Here's an example log4j.properties I tried using, but it didn't do anything (admittedly, I'm not familiar with log4j). I also tried changing the `log4j.appender.stdout.layout.ConversionPattern` to see if I could get anything to happen with log4j.properties, but it still had no impact on the logging output. Any tips or something I'm missing about logging for SimianArmy?

```
     log4j.rootLogger=INFO, stdout, R

     # stdout
     log4j.appender.stdout=org.apache.log4j.ConsoleAppender
     log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
     log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} - %-5p %C{1} - [%F:%L] %m%n
     
     # file
     log4j.appender.R=org.apache.log4j.RollingFileAppender
     log4j.appender.R.File=/var/log/simianarmy/simianarmy.log
     log4j.appender.R.MaxFileSize=900KB
     # Keep one backup file
     log4j.appender.R.MaxBackupIndex=3
     
     log4j.appender.R.layout=org.apache.log4j.PatternLayout
     log4j.appender.R.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} - %-5p %C{1} - [%F:%L] %m%n
```

I tried using java 1.6 but that seems to be a non-starter (error below). Is there anyone who has tried modifying log4j.properties or logging it to a file? I've spent a fair amount of time on it, different permutations, simple minimally viable cases to confirm it is an issue with SimianArmy and not my setup. Hopefully that gives you what you need to fix it - I'm just going to have my /etc/init.d/simianarmy script redirect stdout/stderr to a file and logrotate it.

```
FAILURE: Build failed with an exception.

* What went wrong:
nebula/plugin/netflixossproject/NetflixOssProjectPlugin : Unsupported major.minor version 51.0

```

--- a/build.gradle
+++ b/build.gradle
@@ -15,7 +15,7 @@ ext.githubProjectName = 'SimianArmy'
 group = "com.netflix.${project.name}"
 apply plugin:'eclipse-wtp'
 
-repositories { 
+repositories {
     mavenLocal()
     mavenCentral()
 }
@@ -47,7 +47,6 @@ dependencies {
     compile 'org.apache.jclouds.driver:jclouds-slf4j:1.9.0'
     compile 'org.apache.jclouds.api:ec2:1.9.0'
     compile 'org.apache.jclouds.provider:aws-ec2:1.9.0'
-    compile 'ch.qos.logback:logback-classic:1.0.13'
     compile 'com.netflix.servo:servo-core:0.9.4'
 
     testCompile 'org.testng:testng:6.3.1'

